textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordPhraseTest.java

```diff
@@ -271,6 +271,7 @@ public void testWordInMultipleFieldsQueryWithStopWords3() throws Exception {
         for (ITuple tuple : keywordTestConstants.getSampleMedlineRecord()) {
             MedDataWriter.insertTuple(tuple);
         }
+        MedDataWriter.close();
```

Comment:
@chenlica @qinhuah Please do not bother with all `MedDataWriter` cases. I recently finished a PR to refactor all KeywordMatcherTests. It seems like this is a test that I missed in my refactoring. I'll create a new PR to get rid of the `MedDataWriter` stuff. So please don't touch it now to avoid conflicts.